.github/workflows/swift-toolchain.yml

```diff
@@ -692,7 +692,10 @@ jobs:
           ref: ${{ inputs.swift_revision }}
           path: ${{ github.workspace }}/SourceCache/swift
           show-progress: false
-
+      - name: Setup python
```

Comment:
We should have this upstream. Is there a reason why this cannot be upstream?

Reply:
upstream branch merged